src/routes/(console)/account/organizations/+page.svelte

```diff
@@ -104,16 +128,17 @@
                     <svelte:fragment slot="status">
                         {#if isCloudOrg(organization)}
                             {#if isNonPayingOrganization(organization)}
-                                <Tooltip>
-                                    <Badge
-                                        size="xs"
-                                        variant="secondary"
-                                        content={tierToPlan(organization?.billingPlan)?.name} />
-
-                                    <span slot="tooltip">
-                                        You are limited to 1 free organization per account
-                                    </span>
-                                </Tooltip>
+                                {#if planName}
+                                    {#await planName then name}
```

Comment:
won't there be a slight content/layout shift until this loads and if the api call was made?